scripts/ci/run-jenkins.sh

```diff
@@ -40,7 +40,10 @@ if [[ ${CI_TAGS} == *'osx-amd64'* ]]; then CFLAGS="$CFLAGS -m64 -arch x86_64"; L
 if [[ ${CI_TAGS} == *'win-i386'* ]]; then PLATFORM=Win32; EXTRA_CONF_FLAGS="${EXTRA_CONF_FLAGS} --host=i686-w64-mingw32"; export MONO_EXECUTABLE="${MONO_REPO_ROOT}/msvc/build/sgen/Win32/bin/Release/mono-sgen.exe"; fi
 if [[ ${CI_TAGS} == *'win-amd64'* ]]; then PLATFORM=x64; EXTRA_CONF_FLAGS="${EXTRA_CONF_FLAGS} --host=x86_64-w64-mingw32 --disable-boehm"; export MONO_EXECUTABLE="${MONO_REPO_ROOT}/msvc/build/sgen/x64/bin/Release/mono-sgen.exe"; fi
 
-if [[ ${CI_TAGS} == *'coop-gc'* ]]; then EXTRA_CONF_FLAGS="${EXTRA_CONF_FLAGS} --with-cooperative-gc=yes"; fi
+if   [[ ${CI_TAGS} == *'coop-gc'* ]]; then EXTRA_CONF_FLAGS="${EXTRA_CONF_FLAGS} --with-cooperative-suspend=yes";
```

Comment:
why do we still need this?

Reply:
The only way we can merge this without breaking existing CI is by _adding_ the `coop-suspend` CI_TAGS since earlier branches will still need the `coop-gc` CI_TAGS (remember we only have one config for PR builds) until we no longer care about them.

This means that we don't need the old check since we'd have the new one already.

Reply:
Fair enough, let me remove it then.

Reply:
I fixed it on CI and we are now passing both `coop-gc` and `coop-suspend`.

Reply:
the `${CI_TAGS} == *'coop-gc'*` check is still there :)

Reply:
Btw. I assume having both `coop-suspend` and `hybrid-suspend` is a valid case right? We need to do separate `if` instead of `if-else` then.

Reply:
@akoeplinger the push was failing due to a github issue, but should be there now. And having both `coop-suspend` and `hybrid-suspend` at the same time is not valid, because you either use cooperative suspended or hybrid suspend, not both. Let me add this contraint to `configure.ac`.